packages/sirius-web/backend/sirius-web-persistence/src/main/resources/db/changelog/sirius-web-schema.sql

```diff
@@ -16,7 +16,7 @@ CREATE TABLE idmapping (
 CREATE TABLE project (
     id uuid DEFAULT gen_random_uuid() NOT NULL,
     name text NOT NULL,
-    CONSTRAINT project_name_length CHECK (((char_length(name) > 0) AND (char_length(name) <= 50)))
+    CONSTRAINT project_name_length CHECK (((char_length(name) > 0) AND (char_length(name) <= 1024)))
```

Comment:
Inconsistent with `ProjectService.isValidProjectName(String)` and `isNameInvalid` in `RenameProjectModalMachine`. Also, as for the representations I'm not sure our current UI would react well to actually long project names (no need to go as far as 1024).

Reply:
The previous value was also inconsistent.
The only goal here is to limit the risk to fall to the SQL error raised in https://github.com/eclipse-sirius/sirius-components/issues/1775.